src/lib/Components/Consignments/__stories__/Consignments.story.tsx

```diff
@@ -31,7 +31,8 @@ const blankMetadata = {
 storiesOf("Consignments/_Screens")
   .add("Consignments", () => <Nav />)
   .add("Welcome Page", () => <Welcome navigator={nav} route={route} />)
-  .add("Overview Page", () => <Overview navigator={nav} route={route} setup={{}} />)
+  .add("Overview Page (Fresh)", () => <Overview navigator={nav} route={route} setup={{}} />)
+  .add("Overview Page", () => <Overview navigator={nav} route={route} setup={null} />)
```

Comment:
This one will always be blank, and the other will always try restore the last version